server/server.go

```diff
@@ -0,0 +1,467 @@
+package server
```

Comment:
Perhaps the usage is better if this is in the root? Just like the `http` package etc?

```go
srv := server.Server{
    [...]
}
// Vs
srv := epplib.Server{
   [...]
```

WDYT?